src/extensions/layout-selector/editor.scss

```diff
@@ -33,29 +33,54 @@
 }
 
 .coblocks-layout-selector-modal {
+	background-color: #f5f5f5;
 	height: 85vh;
 	position: relative;
 	width: 1200px;
 
 	.components-modal__content {
 		height: auto;
+		padding: 0;
 	}
 
 	.components-modal__header {
 		border-bottom: none;
 		height: 96px;
-		padding: 0 24px;
+		margin: 0;
+		padding: 0;
```

Comment:
I would only force `padding-left: 0` here otherwise the close button is hugging the side of the modal window.

Reply:
A slight difference here with the placement of the close button. I personally prefer the right aligned button. Any thoughts @richtabor?

**with `padding: 0`**
![image](https://user-images.githubusercontent.com/30462574/98576520-e007eb00-2277-11eb-9e88-14794feb7556.png)

**with `padding-left: 0`**
![image](https://user-images.githubusercontent.com/30462574/98576561-f0b86100-2277-11eb-896f-c530d5abc7bf.png)

Reply:
Let's keep it as is.